winrt/lib/text/CanvasFontFace.abi.idl

```diff
@@ -175,6 +187,24 @@ namespace Microsoft.Graphics.Canvas.Text
             [out] UINT32* outputCount,
             [out, size_is(, *outputCount), retval] int** outputElements);
 
+        HRESULT GetGlyphMetrics(
+          [in] UINT32 inputCount,
+          [in, size_is(inputCount)] UINT16* inputElements,
+          [in] boolean isSideways,
+          [out] UINT32* outputCount,
+          [out, size_is(, *outputCount), retval] CanvasGlyphMetrics** outputElements);
+
+        HRESULT GetGdiCompatibleGlyphMetrics(
+          [in] float emSize,
+          [in] float pixelsPerDip,
```

Comment:
Elsewhere, we expose a DPI parameter, and convert it to/from pixels-per-dip. Can we make this match?